### PR TITLE
Fixed ui offsets, pronounced the anchors a little more

### PR DESCRIPTION
**Pull request by Telzhaak**
^ title

Changelog needed?

### CODE REVIEW

File: examples/assets/ui/example.ron [hunk 1]
```diff
@@ -180,9 +180,9 @@ Container(
         Text(
             transform: (
                 id: "TopLeft",
-                y: -100.0,
-                width: 500.,
-                height: 75.,
+                y: 100.0,
```
Comment (jojolepro) [inline examples/assets/ui/example.ron]:
```suggestion
                y: 180.0,
```

File: examples/assets/ui/example.ron [hunk 2]
```diff
@@ -214,9 +214,9 @@ Container(
         Text(
             transform: (
                 id: "BottomRight",
-                y: -300.0,
-                width: 500.,
-                height: 75.,
+                y: -100.0,
```
Comment (jojolepro) [inline examples/assets/ui/example.ron]:
```suggestion
                y: -180.0,
```

**Comment by Telzhaak** [comment]
This PR is obsolete, since @Moxinilian has incorporated these changes in PR #1064,
therefore closed.

**Comment by jojolepro** [comment]
Thanks for opening the pr :)

**Pull request closed without merging**